haiku-apps/lingua/lingua-2.0.0.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A multiple language translation utility"
+DESCRIPTION="Lingua is a multiple language translation utility. \
+It translates simple words and phrases to and from English \
+into several languages, including Latin, German, Dutch, Italian, French etc."
+HOMEPAGE="https://github.com/HaikuArchives/Lingua"
+COPYRIGHT="The Aevum Network authors"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="246414192994f9ef8f7d1afcaa705b5e80045c7b"
+SOURCE_URI="https://github.com/HaikuArchives/Lingua/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1c1b68d219c44b0f3df33246169211057dd67d62d2b5117c2effcbb984161c5a"
+SOURCE_FILENAME="Lingua-$srcGitRev.tar.gz"
+SOURCE_DIR="Lingua-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 !x86_64"
```

Comment:
If you want to keep x86 as primary architecture here mark it as broken (no way to test it) or remove it from the line:
```diff
-ARCHITECTURES="x86_gcc2 x86 !x86_64"
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
```